### ISSUE TITLE
[Translation tool] it

### ISSUE THREAD

**Issue by Tizianoz93** [briannesbitt/Carbon#1630]
|English|it|comments|
|--|--|--|
|2yrs|2 a.|2 anni because 2 a. abbreviation is not used in italy|
|2mos|2 mes.|2 mesi because 2 mes. is not used in italy and the lenght is the same|
|2d|2 g.|2 gg|
|2h|2 o.|2h because it's immutable from english to italian and 2 o. is not used in italy|
|Monday|lunedì|Lunedì|
|Mon|lun|Lun|
|Mo|lu|Lu|
|Tuesday|martedì|Martedì|
|Tue|mar|Mar|
|Tu|ma|Mar (ma is not used in italy)|
|Wednesday|mercoledì|Mercoledì|
|Wed|mer|Mer|
|We|me|Mer (me is not used in italy)|
|Thursday|giovedì|Giovedì|
|Thu|gio|Gio|
|Th|gi|Gio (gi is not used in italy)|
|Friday|venerdì|Venerdì|
|Fri|ven|Ven|
|Fr|ve|Ven (ve is not used in italy)|
|Saturday|sabato|Sabato|
|Sat|sab|Sab|
|Sa|sa|Sab (sa is not used in italy)|
|Sunday|domenica|Domenica|
|Sun|dom|Dom|
|Su|do|Dom (do is not used in italy)|
|January|gennaio|Gennaio|
|Jan|gen|Gen|
|February|febbraio|Febbraio|
|Feb|feb|Feb|
|March|marzo|Marzo|
|Mar|mar|Mar|
|April|aprile|Aprile|
|Apr|apr|Apr|
|May|maggio|Maggio|
|May|mag|Mag|
|June|giugno|Giugno|
|Jun|giu|Giu|
|July|luglio|Luglio|
|Jul|lug|Lug|
|August|agosto|Agosto|
|Aug|ago|Ago|
|September|settembre|Settembre|
|Sep|set|Set|
|October|ottobre|Ottobre|
|Oct|ott|Ott|
|November|novembre|Novembre|
|Nov|nov|Nov|
|December|dicembre|Dicembre|
|Dec|dic|Dic|

**Comment by kylekatarnls** [comment]
Hello, thanks for this comments. I will inspect deeply the short units.

About Month/Day uppercase, as far as I know, it's wrong in Italian, by default they are lowercase `Ci vediamo lunedì`, uppercase is a English exception: `See you Monday` (even in the middle of a sentence).

If a month/day is needed in a title, a sentence beginning, `ucfirst()` should be used on it.

Are you sure there is no distinction possible between shot and minimalist formats?

For example short format:
```php
echo Carbon::now()->isoFormat('llll');
```
Outputs 3-letters abbreviations in English:
```
Wed, Feb 27, 2019 1:01 PM
```

Minimalist format (2-letters in English) can be used to display a table calendar:

|Su|Mo|Tu|We|Th|Fr|Sa|
|-|-|-|-|-|-|-|
||||||1|2|
|3|4|5|6|7|8|9|
|10|11|12|13|14|15|16|
|17|18|19|20|21|22|23|
|24|25|26|27|28|||

Are you sure `Mar`, `Gio`, etc. will sweet best for both?

**Comment by kylekatarnls** [comment]
Here on Wikipedia, I read "Symbolo d" for Giorno https://it.wikipedia.org/wiki/Giorno
It seems consistent with ora = h.

Does it seem right to you. And an other question: should we have a space between the number and the unit?

**Comment by kylekatarnls** [comment]
Short unit released in 2.14.1, months to be confirmed.

**Comment by kylekatarnls** [comment]
I googled "Italian calendar" and most of pictures I found are 2-letters calendars:
https://www.vectorstock.com/royalty-free-vector/italian-2018-year-calendar-vector-12475133
https://www.canstockphoto.ch/kreis-vektor-2018-kalender-43638910.html

So I will keep "min" months variants.

If you have more references, please feel free to add them.
